support/notulen-utils.js

```diff
@@ -79,11 +79,11 @@ export async function ensureVersionedNotulen(meeting, treatments, kind, publicTr
     console.log(`Creating a new versioned notulen of kind ${kind} for ${meeting.uri}`);
     let html;
     if (kind === NOTULEN_KIND_FULL) {
-      const data = await buildDataForMeetingNotes({meeting, treatments, allPublic: true});
+      const data = await buildDataForMeetingNotes({meeting, treatments, previewType: false, allPublic: true});
```

Comment:
bit confused since the constants have string values now, so I assume this still needs to change?